tests/fullstack-test/expr/regexp.test

```diff
@@ -94,3 +94,9 @@ mysql> set tidb_enforce_mpp=1; select regexp_like(data, pattern, match_type) as
 |    1 |
 |    1 |
 +------+
+
+mysql> drop table if exists test.t;
+mysql> create table test.t (data varchar(30), pattern varchar(30), pos int, occur int, ret_op int, match_type varchar(30));
+mysql> alter table test.t set tiflash replica 1;
+func> wait_table test t
+mysql> set tidb_enforce_mpp=1; select regexp_instr("1", "1", pos, occur, 1, match_type) as res from test.t;
```

Comment:
For empty input, should test the case that no error throw even if `pattern` is empty string.

Reply:
And also add a test that with non-empty input.

Reply:
> 
> For empty input, should test the case that no error throw even if `pattern` is empty string.

Done